src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts

```diff
@@ -100,7 +100,8 @@ registerActiveXtermAction({
 	title: localize2('focusTerminalInput', 'Focus Terminal Input'),
 	keybinding: {
 		primary: KeyMod.CtrlCmd | KeyCode.UpArrow,
-		when: TerminalChatContextKeys.focused,
+		secondary: [KeyMod.CtrlCmd | KeyCode.KeyI],
+		when: ContextKeyExpr.and(TerminalChatContextKeys.focused, CTX_INLINE_CHAT_FOCUSED.toNegated()),
 		weight: KeybindingWeight.WorkbenchContrib,
```

Comment:
I'm surprised this works are the regular ctrl/cmd+i keybinding above has a weight of `weight: KeybindingWeight.ExternalExtension + 1`?

Reply:
the conflicting action is here 
https://github.com/microsoft/vscode/blob/9ed28b5d674d01f475c98dd6a77139aaa838f291/src/vs/workbench/contrib/chat/electron-sandbox/actions/voiceChatActions.ts#L521

Reply:
that's for when starting the terminal chat vs this is when it's already open/focused